dominode-bootstrapper/pyproject.toml

```diff
@@ -20,6 +20,7 @@ minio = "^5.0.10"
 
 [tool.poetry.scripts]
 dominode-admin = 'dominode_bootstrapper.dominodeadmin:app'
+geonode-admin = 'dominode_bootstrapper.geonodeadmin:app'
```

Comment:
Please remove this line

We do not want to have multiple admin scripts, just a single one. As it is now if you call `poetry run dominode-admin geonode` you can already access all cli commands of the `geonodeadmin.py` module. No need for any extra work